cpp/CMakeLists.txt

```diff
@@ -788,8 +788,7 @@ if(CUDF_BUILD_BENCHMARKS)
   # do not support the latest version of fmt, which is automatically pulled into our conda
```

Comment:
Missed this the first time, can you move the following over to the function inside `get_nvbench`:
```cmake
  # do not support the latest version of fmt, which is automatically pulled into our conda
  # environments by mamba.
  set(CPM_DOWNLOAD_fmt TRUE)
```